book/page_creation.rst

```diff
@@ -2,207 +2,197 @@
-provides the ``dev`` environment, and ``app.php`` provides the ``prod`` environment.
-All web accesses to Symfony normally go through one of these front controllers.
-(The ``test`` environment is normally only used when running unit tests, and so
-doesn't have a dedicated front controller. The console tool also provides a
-front controller that can be used with any environment.)
-
-When the front controller initializes the kernel, it provides two parameters:
-the environment, and also whether the kernel should run in debug mode.
-To make your application respond faster, Symfony maintains a cache under the
-``app/cache/`` directory. When debug mode is enabled (such as ``app_dev.php``
-does by default), this cache is flushed automatically whenever you make changes
-to any code or configuration. When running in debug mode, Symfony runs
-slower, but your changes are reflected without having to manually clear the
-cache.
+    Before continuing, make sure you've read the :doc:`Installation </book/installation>`
+    chapter and can access your new Symfony app in the browser.
 
-.. index::
-   single: Page creation; Example
+Suppose you want to create a page - ``/lucky/number`` - that generates a
+lucky (well, random) number and prints it. To do that, create a class and
+a method inside of it that will be executed when someone goes to ``/lucky/random``::
 
-The "Random Number" Page
-------------------------
+    // src/AppBundle/Controller/LuckyController.php
+    namespace AppBundle\Controller;
 
-In this chapter, you'll develop an application that can generate random numbers.
-When you're finished, the user will be able to get a random number between ``1``
-and the upper limit set by the URL:
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+    use Symfony\Component\HttpFoundation\Response;
 
-.. code-block:: text
+    class LuckyController
+    {
+        /**
+         * @Route("/lucky/number")
+         */
+        public function numberAction()
+        {
+            $number = rand(0, 100);
 
-    http://localhost/app_dev.php/random/100
+            return new Response(
+                '<html><body>Lucky number: '.$number.'</body></html>'
+            );
+        }
+    }
 
-Actually, you'll be able to replace ``100`` with any other number to generate
-numbers up to that upper limit. To create the page, follow the simple two-step
-process.
+Before diving into this, test it out!
 
-.. note::
+.. code-block:: text
 
-    The tutorial assumes that you've already downloaded Symfony and configured
-    your webserver. The above URL assumes that ``localhost`` points to the
-    ``web`` directory of your new Symfony project. For detailed information
-    on this process, see the documentation on the web server you are using.
-    Here are some relevant documentation pages for the web server you might be using:
+    http://localhost:8000/app_dev.php/lucky/number
```

Comment:
Why a text box? If we add it to the text, people can click on it and see it directly (otherwise, I would prefer an inline code block)